examples/experiment_specs/nell/new_features/standard/tuning/baseline_vs_l1-0.005_l2-0.001.json

```diff
@@ -2,31 +2,24 @@ load new_feature_experiment_base
 {
   "split": "nell_with_negatives",
   "operation": {
-    "mode": "learn models",
+    
     "features": {
+	"type": "pra",
       "path finder": {
+	"type": "RandomWalkPathFinder",
         "path type factory": {
   "name": "VectorPathTypeFactory",
   "spikiness": 3,
   "reset weight": 0.25,
-  "embeddings": "pca_svo"
-},
-
-        "walks per source": 100,
-        "path finding iterations": 3,
-        "path accept policy": "paired-only"
-      },
-      "path selector": {
-        "number of paths to keep": 1000
-      },
-      "path follower": {
-        "walks per path": 50,
-        "matrix accept policy": "paired-targets-only"
+  "embeddings": "/tienan/pra/examples/embeddings/pca_svo/embeddings.tsv"
```

Comment:
Can you remove your machine-specific path, here?  Something like `/embedding_dir/pca_svo/embeddings.tsv` would be better - it's more obvious that a user is intended to change the path this way.  Also, the code used to not require you to give a full path...  Not sure if you can just use "pca_svo" here any more or not.  It's been quite a while since I touched this code.